cpp/cmake_modules/FindabslAlt.cmake

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+if(abslAlt_FOUND)
+  return()
+endif()
+
+set(find_package_args)
+
+if(abslAlt_FIND_VERSION)
+  list(APPEND find_package_args ${abslAlt_FIND_VERSION})
+endif()
```

Comment:
Do we need this?

Reply:
As we use it currently not, I guess it could produce confusing results, let's remove it.